checker/md5sum-checker.cwl

```diff
@@ -7,7 +7,7 @@ cwlVersion: v1.0
 
 requirements:
 - class: DockerRequirement
-  dockerPull: quay.io/agduncan94/checker-md5sum
+  dockerPull: quay.io/dockstore-testing/md5sum-checker:feature_emlys_DOC-654
```

Comment:
This is a chicken and egg situation, but I think you want the version to be `1.0.0`, which will only exist after this PR is merged and you have created a 1.0.0 GitHub release. Hmm, but then your tests will fail.

Maybe Gary's self-described "hacky" idea in another comment is the way to go.